packages/react-components/react-accordion/src/index.ts

```diff
@@ -40,10 +40,12 @@ export type {
 export {
   AccordionHeader,
   accordionHeaderClassNames,
+  AccordionHeaderContextProvider,
   renderAccordionHeader_unstable,
   useAccordionHeaderContextValues_unstable,
   useAccordionHeaderStyles_unstable,
   useAccordionHeader_unstable,
+  useAccordionHeaderContext,
```

Comment:
I think all our `useX` hooks have the `_unstable` suffix by convention. +1 to the export, but I think it needs to be renamed first

Reply:
the "_unstable" suffix is a way of telling consumers that the API might change in the future, we're not expecting this to happen here as the Accordion has been stable for a long time now :)

Reply:
Our `_unstable` convention has change recently https://github.com/microsoft/fluentui/wiki/new-release-process---v9-packages#is-there-still-a-need-for-_unstable-suffixes-for-hooksutilities